www/js/controllers.js

```diff
@@ -1,5 +1,21 @@
 angular.module('building-blocks.controllers', [])
 
 .controller('HomeController', function($scope, News) {
-   $scope.news = News.query();
+  $scope.news = News.query();
+})
+
+.controller('MessageController', function($scope, HelpRequest) {
+  $scope.error = null;
+  $scope.help_request = {};
+
+  $scope.createHelpRequest = function() {
+    HelpRequest.save($scope.help_request, function(response){
+      console.log(response);
+      $scope.message = response.message;
+      // Move on to another page or whatever you want to do
```

Comment:
you will get a thank you message, I just updated the screenshot... have a look @tochman